### DISCUSSION TITLE
关于NCheck-Box中无法显示勾选的问题

### DISCUSSION THREAD

**Discussion by elic-hwz** [tusen-ai/naive-ui#5808] [Q&A] [answered]
首先数据已经初始化完成，
![Screenshot_20240322_184459](https://github.com/tusen-ai/naive-ui/assets/74193460/f27d34fc-c58b-4746-b85e-b8294f161a0c)
这是template里面的ncheckbox配置
![Screenshot_20240322_184645](https://github.com/tusen-ai/naive-ui/assets/74193460/21914835-b0ca-45fd-9cc0-2cb3dc870347)
现在渲染页面后，页面不显示勾选状态，按理来说应该会将数组里有的值勾选上去
![Screenshot_20240322_185128](https://github.com/tusen-ai/naive-ui/assets/74193460/65bc464e-c168-46e2-9be0-e4f8f7ad3337)

**Comment by jahnli** [comment, marked as answer]
去掉 v-model 换成 :value + @update:value 试试，或者单纯使用 v-model

**Reply by elic-hwz** [reply]
已经解决，需要将v-model改成v-model:value="redistribute"，并且去掉update函数，因为v-model本身具有双向绑定的特性
![Screenshot_20240426_160058](https://github.com/tusen-ai/naive-ui/assets/74193460/0c555fda-c7fb-4e8d-9df8-70bd9ed338f0)
![Screenshot_20240426_160112](https://github.com/tusen-ai/naive-ui/assets/74193460/283932c8-29bc-4d62-9063-890e47379f51)
![Screenshot_20240426_160138](https://github.com/tusen-ai/naive-ui/assets/74193460/99ea076c-474e-446d-99da-8736342f7972)
